packages/core/src/connectors/injected.ts

```diff
@@ -171,39 +168,42 @@ export class InjectedConnector extends Connector {
   }
 
   async account(): Promise<AccountInterface> {
-    /*
     this.ensureWallet();
 
-    if (!this._wallet || !this._wallet.account) {
+    if (!this._wallet) {
       throw new ConnectorNotConnectedError();
     }
 
-    return this._wallet.account;
-    */
-    throw new Error("TODO");
+    // Temporary workaround since StarknetWindowObject type isn't correct
+    const temp = this._wallet as any;
+
+    if (!temp.account) {
+      throw new ConnectorNotConnectedError();
+    }
+
+    return temp.account as AccountInterface;
```

Comment:
I believe the long term solution for this is to make a `wallet_requestAccounts` request with `silent_mode: true`, but it looks like the wallet implementation is bugged for now :(

Reply:
Yep, I'll add a comment to keep that in mind for the future